.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+    "trailingComma": "es5",
```

Comment:
It's better to use ES6 instead of ES5, which provides more features for JavaScript and has more lifetime of browser support.

Reply:
You are right, but currently, they have not implemented "trailingComma" for ES6, or the new ES6 standard has not implemented changes to this parameter. The following is the link to the implementation https://github.com/prettier/prettier/tree/main/tests/format/js/trailing-comma